.release-it.json

```diff
@@ -0,0 +1,8 @@
+{
+  "git": {
+    "tagName": "v${version}"
```

Comment:
not strictly needed since the other tags start with a `v` already, but it makes it easier to copy this into new projects.